src/views/v2/tickets/item.php

```diff
@@ -105,7 +105,7 @@
 	'data-ticket-id'      => (string) $ticket->ID,
 	'data-available'      => $this->get( 'data_available' ),
 	'data-has-shared-cap' => $this->get( 'data_has_shared_cap' ),
-	'data-ticket-price'   => $ticket->price,
+	'data-ticket-price'   => (string) $ticket->price,
```

Comment:
We need to set the `@version TBD` for this template, as well as adding the `@since` with the changes that we're introducing. So that there's some sort of tracking of the changes that we're introducing and why.

Reply:
Thanks. Here is a PR for that: https://github.com/the-events-calendar/event-tickets/pull/2080